lxd/db/node.go

```diff
@@ -109,6 +109,23 @@ func (c *ClusterTx) GetNodeByName(name string) (NodeInfo, error) {
 	}
 }
 
+// GetNodeByID returns the node with the given ID.
+func (c *ClusterTx) GetNodeByID(nodeID string) (NodeInfo, error) {
```

Comment:
@monstermunchkin it feels like the `nodeID` argument should be an int64 here to align with the data type used by the database layer when returning the ID. Then if you have a string it can be converted outside of the function.

Reply:
@tomponline Actually, this function can be removed. I believe I previously needed this function but now it's not used anymore.